source/jade/_mixins/user-mixins.jade

```diff
@@ -86,7 +86,7 @@ mixin pwd-reset-form
 mixin alert-login-failed
   if viewModel && viewModel.errorMessage
     .alert(role='alert')
-      #message= 'Error: ' + viewModel.errorMessage
+      #message!= 'Error: ' + viewModel.errorMessage
```

Comment:
@DinisCruz thanks for your feedback! Yeah It makes a lot of sense, I think using the new mixin is a better option.